lib/src/model.dart

```diff
@@ -481,12 +539,10 @@ class Class extends ModelElement implements EnclosedElement {
 
   List<Field> get inheritedProperties {
     if (_inheritedProperties != null) return _inheritedProperties;
-
-    InheritanceManager manager = new InheritanceManager(element.library);
     Map<String, ExecutableElement> cmap =
-        manager.getMembersInheritedFromClasses(element);
+        library.inheritanceManager.getMembersInheritedFromClasses(element);
     Map<String, ExecutableElement> imap =
-        manager.getMembersInheritedFromInterfaces(element);
+        library.inheritanceManager.getMembersInheritedFromInterfaces(element);
 
     _inheritedProperties = [];
     List<ExecutableElement> vs = [];
```

Comment:
I believe `vs` here stands for values. If so, perhaps we should rename the variable for understandability?